apps/src/gamelab/animationListModule.js

```diff
@@ -710,17 +708,8 @@ export function animationSourceUrl(key, props, withVersion = false) {
   }
 
   // 2. Otherwise it's local to this project, and we should use the animation
-  //    key to look it up in the animations API.
-  return animationsApi.basePath(key) + '.png' +
-      (
-        (withVersion && props.version)
-        ? '?version=' + props.version
-        // Temporary: Local-cachebust animation requests, so after restoring
-        // and old version of the project the client isn't seeing newer animations.
-        // Should be able to tear this out once we always request animations
-        // by specific version.
-        : '?cachebust=' + Math.floor(Math.random() * 0xFFFFFF).toString(16)
```

Comment:
The cachebust parameter is no longer needed.

The [original purpose](https://github.com/code-dot-org/code-dot-org/pull/21259/commits/dbfc5d0a1c9fa04e57aa6612e6755a44631a77c8) was to fix version restore, where `/v3/animations/<my-project>/animation1` should give a different result before the restore and after it, but the client was locally caching the pre-restore version.  Now that we are always appending a version id, the client will know to grab the new version post-restore.

We're fine in the falsy version case because our restore operation repairs manifests - post-restore, all the version ids will be present and valid.